[IMP] Fiscal Localizations: Add CIS deduction info for UK (#11451)
odoo/documentation: content/applications/finance/fiscal_localizations/united_kingdom.rst

        - Allows sending the tax report via the MTD-VAT API to HMRC.
   * - :guilabel:`UK BACS Payment Files`
     - `l10n_uk_bacs`
     - Allows generating :ref:`localization/united-kingdom/BACS-files` for bill and invoice payments
   * - :guilabel:`UK - Construction Industry Scheme`
     - `l10n_uk_reports_cis`
     -  - Allows sending the Monthly return to |HMRC|
        - CIS Deduction (GB) report for UK construction industry
   * - :guilabel:`UK - HMRC API`
     - `l10n_uk_hmrc`

>      -  - Includes the basics of |HMRC|.

Reviewer: ```suggestion
     -  Includes the basics of |HMRC|.
```
isn't this CIS-specific (not sure what else the "basics of HRMC" entail but we might want to play it safe here? I would also add a link to that section in the doc

   :guilabel:`Manage Users`. Select the user who is connected to HMRC.
#. Go to the :guilabel:`UK HMRC Integration` tab and click :guilabel:`Reset Authentication
   Credentials` or :guilabel:`Remove Authentication Credentials` button.

> #. You can now :ref:`register your company to HMRC <localization/united-kingdom/hmrc-registration>`
>    and submit the tax report for this company.

Reviewer: ```suggestion
#. :ref:`Register your company to HMRC <localization/united-kingdom/hmrc-registration>`
   and submit the company's tax report.
```

each subcontractor is registered under the |CIS|. They are also responsible for maintaining records
of all payments and deductions. Additionally, contractors must submit monthly returns to HMRC,
including the following details:

- Information about the subcontractors.
- Records of payments made and any deductions applied.
- A declaration confirming that the employment status of all subcontractors has been reviewed.
- A declaration confirming that all subcontractors requiring verification have been verified.

.. note::
   If no payments were made to subcontractors in the previous tax month, contractors must notify
   |HMRC| by the 19th of the month to avoid a penalty.

.. _localization/united-kingdom/cis-configuration:

Configuration
-------------

:ref:`Install <general/install>` the :guilabel:`UK - Construction Industry Scheme` module.

.. list-table::
   :header-rows: 1

   * - Name
     - Technical name
     - Description
   * - :guilabel:`UK - Construction Industry Scheme`
     - `l10n_uk_reports_cis`
     -  - Allows sending the Monthly return to |HMRC|
        - CIS Deduction (GB) report for UK construction industry
   * - :guilabel:`UK - HMRC API`
     - `l10n_uk_hmrc`

>      -  - Includes the basics of |HMRC|.

Author: @auva-odoo Yes, indeed! I really was in a "CIS bubble" there 😳😜

.. note::
   Installing the :guilabel:`UK - Construction Industry Scheme` module installs the other module
   simultaneously.

To enable the :guilabel:`Test` mode and use test credentials, activate
:ref:`developer mode <developer-mode>` and go to :menuselection:`Settings --> Technical -->
System Parameters`. Then search for `l10n_uk_hmrc.api_mode`. Select it and switch the value from
`production` to `test`.

.. _localization/united-kingdom/cis-monthly-returns:

Monthly returns
---------------

Monthly returns only work for vendor bills and vendor refunds. To submit a complete return to
|HMRC|, several steps must be followed to report all payments made to subcontractors under the
scheme during the previous tax month:

- :ref:`localization/united-kingdom/cis-contractor-setup`
- :ref:`localization/united-kingdom/cis-subcontractor-setup`
- :ref:`localization/united-kingdom/cis-vendorbills`
- :ref:`localization/united-kingdom/cis-monthly-return-sending`

.. _localization/united-kingdom/cis-contractor-setup:

Contractor setup
~~~~~~~~~~~~~~~~

To configure the company |HMRC| information, go to the :guilabel:`Settings` app. In the

> :guilabel:`Companies` section, click :guilabel:`Manage Companies` and select the company. Open the

Author: @auva-odoo You're right, much easier 🤪😂

:guilabel:`HMRC` tab and configure the information in the :guilabel:`HMRC Credentials` and the
:guilabel:`Contractor details` sections. All the fields are mandatory.

.. _localization/united-kingdom/cis-subcontractor-setup:

Subcontractor setup
~~~~~~~~~~~~~~~~~~~

> In the :guilabel:`Contacts` app, open the subcontractor's contact form and select the

Reviewer: I would remove "In the Contacts app" since we could access the contact's form from the Accounting app (e.g. though Accounting --> Vendors --> Vendors)
```suggestion
Access the subcontractor's contact form and select the
```

:guilabel:`Accounting` tab. In the :guilabel:`HMRC Details` section, enable the
:guilabel:`Construction Industry Scheme` option to unlock the additional fields.

> By default, the :guilabel:`Deduction rate` is set to 30%. To modify it, a
> :guilabel:`Verification Number` is required. The verification number is provided by |HMRC| when
> verifying the contact. Make sure to check the status of the subcontractor and update the
> :guilabel:`Deduction Rate` accordingly.

Reviewer: ```suggestion
By default, the :guilabel:`Deduction rate` is set to 30%. To modify it, first enter the
:guilabel:`Verification Number` provided by |HMRC| when
verifying the subcontractor's status, then update the :guilabel:`Deduction Rate` accordingly.
```
just a suggestion, what do you think? (I was thinking we might avoid repeating *verification number*, especially since *verify* comes immediately after it.).

> The :guilabel:`Forename` and :guilabel:`Surname` fields are mandatory if the contact is set to
> :guilabel:`Individual`.

Reviewer: ```suggestion
The :guilabel:`Forename` and :guilabel:`Surname` fields are mandatory if the contact type is set to
:guilabel:`Individual`.
```
and I feel like maybe this could be a note?

.. _localization/united-kingdom/cis-vendorbills:

Vendor bills
~~~~~~~~~~~~

> Additional |CIS| tax must be applied to labour line items on vendor bills. Material line items
> on vendor bills do not apply. The tax rate corresponds to the subcontractor's
> :guilabel:`deduction rate`: :guilabel:`O% CIS`, :guilabel:`20% CIS` or :guilabel:`30% CIS`. To apply
> this, in the :guilabel:`Invoice Lines` section of the vendor bill, select the appropriate |CIS| tax
> rate in the :guilabel:`Taxes` column of the labour line items.

Reviewer: ```suggestion
The appropriate |CIS| tax must be applied to **labor items** on vendor bills based on the subcontractor's
:guilabel:`Deduction Rate`: :guilabel:`0% CIS`, :guilabel:`20% CIS` or :guilabel:`30% CIS`. To apply
the rate, go the :guilabel:`Invoice Lines` section of the vendor bill and select the appropriate |CIS| tax
rate in the :guilabel:`Taxes` column of the **labor** items.
```
- deduction rate: because it's a field label
- rephrasing: to try and minimize repeating tax rate
- labor: US spelling
- I added bold to emphasize the fact that this is necessary for labor items but you could also add a list item in the note below to repeat that *The CIS tax rate is not necessary for material items on vendor bills.* if you think it's necessary.

Author: @auva-odoo Much better like this, thanks! I thought I should use "labour" since it's a UK page ;) 
It is a great idea to add a list item in the note 👍

.. note::
   - When creating a vendor bill, if the :guilabel:`Construction Industry Scheme` option hasn't been
     enabled in the :ref:`subcontractor <localization/united-kingdom/cis-subcontractor-setup>`'s
     :guilabel:`Contact` form, a yellow warning banner appears at the top of the page.
   - If the |CIS| tax used in the vendor bill does not match the expected |CIS| deduction rate for a
     :ref:`subcontractor <localization/united-kingdom/cis-subcontractor-setup>`, a yellow warning

>      banner appears at the top of the page.

Reviewer: Could we maybe group these two to avoid repeating "a yellow warning banner appears at the top of the page" twice?

For example:

A yellow banner appears at the top of the page if:
- A
- B

(just an idea)

     verification have been verified.
   - :guilabel:`Inactivity Indicator`: To declare temporary inactivity.

#. In the :guilabel:`Information correct declaration` section, confirm that the information is true
   and complete by checking the box. Then, enter the :guilabel:`Password` used in the
   :guilabel:`HMRC Credentials` section during
   :ref:`contractor setup <localization/united-kingdom/cis-contractor-setup>`.
#. Click :guilabel:`Send` to prompt Odoo to request |HMRC| to initiate the transaction.

When the transaction results in a response from |HMRC|, Odoo automatically emails the user who
submitted the transaction, informing them that the response from |HMRC| is available in the chatter
section of the company page. If an error has been detected in the submission, a new submission will
be required to ensure compliance with |HMRC| requirements.

The transaction response from |HMRC| appears in the company chatter with an attached
:guilabel:`XML` document. The message specifies the relevant period. To download the
:guilabel:`XML` document, click on the :icon:`fa-download` :guilabel:`(download)` icon. Both the

> electronic and paper versions of the |HMRC| receipt should be retained.

Author: @auva-odoo You're right, it was almost the same info twice. But I think we should keep the fact that Odoo sends an email to the user. I updated it this way. What do you think?

of payments made to subcontractors and forward these deductions to HM Revenue and Customs (HMRC).
These deductions apply only to the labor portion of the payments and serve as advance payments
towards the subcontractor's tax and National Insurance contributions. Contractors are required to
register for the scheme, but subcontractors are not. However, subcontractors who are not registered
face higher payment deductions. Under the |CIS|, contractors must deduct 20% from payments to
registered subcontractors, while the deduction increases to 30% for unregistered ones.

.. seealso::

   - `Construction Industry Scheme (CIS) <https://www.gov.uk/what-is-the-construction-industry-scheme>`_
   - `Guidelines for CIS contractors
     <https://www.gov.uk/what-you-must-do-as-a-cis-contractor>`_
   - `Guidelines for CIS subcontractors
     <https://www.gov.uk/what-you-must-do-as-a-cis-subcontractor>`_

As a contractor, you are required to register with the |CIS| before hiring subcontractors and to
check whether each subcontractor is registered with the |CIS|. You must also maintain records of all
payments and deductions and submit monthly returns to HMRC, including the following details:

> 

Reviewer: nitpicking but since the list items are not full sentence, i would remove the periods?

- Information about the subcontractors.
- Records of payments made and any deductions applied.
- A declaration confirming that the employment status of all subcontractors has been reviewed.
- A declaration confirming that all subcontractors requiring verification have been verified.

.. note::
   If no payments were made to subcontractors in the previous tax month, contractors must notify
   |HMRC| by the 19th of the month to avoid a penalty.

> To submit Monthly Returns to |HMRC|, :ref:`Install <general/install>` the
> :ref:`UK - Construction Industry Scheme <localization/united-kingdom/modules>` module.

Reviewer: ```suggestion
To submit Monthly Returns to |HMRC|, :ref:`install <general/install>` the
:ref:`UK - Construction Industry Scheme <localization/united-kingdom/modules>` module.
```

     been reviewed.
   - :guilabel:`Subcontractor Verification`: To declare that all submitted subcontractors requiring
     verification have been verified.
   - :guilabel:`Inactivity Indicator`: To declare temporary inactivity.

#. In the :guilabel:`Information correct declaration` section, confirm the information is true and
   complete by checking the box. Then, enter the :guilabel:`Password` used in the
   :guilabel:`HMRC Credentials` section during
   :ref:`contractor setup <localization/united-kingdom/cis-contractor-setup>`.
#. Click :guilabel:`Send` to prompt Odoo to request |HMRC| to initiate the transaction.

When |HMRC| replies to a transaction, Odoo automatically notifies the user who submitted it by
email. The email informs them that the response is available in the company's chatter with an
attached XML document for download. Both the electronic and paper versions of the |HMRC| receipt
should be retained. If an error is detected, a new submission is required to comply with |HMRC|
requirements.

.. note::
   - Transactions are updated daily. To manually update the |HMRC| request, click the :icon:`fa-cog`

>      :guilabel:`(cog)` icon and select :guilabel:`Refresh HMRC request`.

Reviewer: ```suggestion
     :guilabel:`(gear)` icon and select :guilabel:`Refresh HMRC request`.
```
